docs/src/mock.md

```diff
@@ -41,31 +41,99 @@ page.route("https://dog.ceo/api/breeds/list/all", handle)
 ```csharp
 await page.RouteAsync("https://dog.ceo/api/breeds/list/all", async route =>
 {
-    var json = /* JsonElement with the test payload */;
+    var json = new { message = new { playwright_dog = new List<string>() } };
     await route.FulfillAsync(new () { Json: json });
 });
 ```
 
 ```java
 page.route("https://dog.ceo/api/breeds/list/all", route -> {
   route.fulfill(new Route.FulfillOptions()
-    .setBody("{\"message\":{\"test_breed\":[]}}"));
+    .setBody("{\"message\":{\"playwright_dog\":[]}}"));
+});
+```
+
+In the example below we intercept the route to the dog API and return a custom response. We then go to that url and assert that the response is our mock data:
```

Comment:
Why do we have both snippets above and snippets below? They seem to do the same thing? Let's have a single snippet and a good explanation for it.

Reply:
Yeah, I meant keeping the full snippet and both explanations. Maybe highlight the `route` (with comments?) to focus the attention on the mocking part.